scripts/WinInstallFiles/InstallJMRI.nsi

```diff
@@ -381,6 +381,41 @@ SetCompressor /SOLID /FINAL lzma
 !include "WordFunc.nsh" ; add header for word manipulation
 !insertmacro VersionCompare ; add function to compare versions
 
+;-- form https://nsis.sourceforge.io/GetInQuotes:_Get_string_from_between_quotes
```

Comment:
I'm on a mobile phone now, but is the word "form" intended? Or should it be "from"?